public/electron.js

```diff
@@ -82,15 +83,31 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  // open devtools if DEV_TOOLS is set to 1
+  if (parseInt(process.env.DEV_TOOLS)) {
```

Comment:
Sure this checks if it's set to 1 :wink:? Regardless, no need to parse this as an int or check for a specific value. Let's check if `PUBLISH50_DEV_TOOLS` (rather than just `DEV_TOOLS`) is set and has a value:

```
if (process.env.PUBLISH50_DEV_TOOLS) {
  // ...
}
```